libvirt/tests/src/libvirt_vcpu_plug_unplug.py

```
@@ -298,6 +300,21 @@ def check_setvcpus_result(cmd_result, expect_error):
             vmxml.set_agent_channel()
         else:
             vmxml.remove_agent_channels()
+        if compat_mode:
+            logging.info("Setting compatibility mode with CPU model: %s", cpu_model)
+
+            try:
+                cpu_xml = vmxml.cpu
+            except xcepts.LibvirtXMLNotFoundError:
```

Ruff (0.15.17)

[error] Undefined name `xcepts`

(F821)